src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1824,14 +1824,19 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         much information as possible will be passed to the compilers.
         -->
 
+    <PropertyGroup>
+      <MSBuildGlobalPropertiesToRemoveFromProjectReferences>%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)</MSBuildGlobalPropertiesToRemoveFromProjectReferences>
```

Comment:
Doesn't this now mean you're collapsing all of the GlobalPropertiesToRemove from each project reference?  IE you would no longer be able to remove a global property from one project reference but not from another one?

Reply:
I think this will retain that behavior because it's within the Target, but I may be wrong about that. I can bring it up in PR reviews.